src/pages/MacKenzieAccessReview.jsx

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import { div, a } from "react-hyperscript-helpers";
+import { Storage } from "../libs/storage";
+const BORDER = "1px solid rgba(0,0,0,0.5)";
+const LOREM_IPSUM =
+  "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.";
+class MacKenzieAccessReview extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = this.initialState();
+  }
+  initialState() {
+    return {
+      currentUser: Storage.getCurrentUser()
+    };
+  }
+  back = () => {
+    const { currentUser } = this.state;
+    const page = currentUser.isChairPerson
+      ? "/chair_console"
+      : currentUser.isMember
+        ? "/member_console"
+        : "/";
+    this.props.history.push(page);
+  };
+  render() {
+    return div(
+      {
+        id: "container", className: 'container-wide centered normal-text primary'
+      },
+      [
+        div(
+          {
+            id: "header",
+            className: 'section',
+            style: {
+              justifyContent: "space-between",
+              alignItems: "center",
+              display: "flex"
+            }
+          },
```

Comment:
Thinking out loud ... This is one approach, which I am totally fine with. Another would be to break out each div that will eventually be a component into a component now. That could be a stub component that's called over and over again with different style and content props. Either way is fine, so no need to tackle this now.